### ISSUE TITLE
Autocomplete dropdown still occurs even when I have selected element from the dropdown.

### ISSUE THREAD

**Issue by lovingyugs** [react-materialize/react-materialize#536]
### Description

In Autocomplete component, even after selecting any matching element from the list the list did not get away, if the list item is a substring in other list items, then the other list items keep on appearing.

What I want is that- If we have selected an item or element from the dropdown list, then the dropdown list should go away, which is not happening in the below example.

### Steps to Reproduce

Check here on-[react-materialize]( https://react-materialize.github.io/#/forms)
You can reproduce the error by following the below steps

1. Replace the Autocomplete component code with this-
````
  <Autocomplete
    title='Company'
    data={
      {
        'Apple': null,
        'Microsoft': null,
        'Google': 'http://placehold.it/250x250',
        'Google again': null,
      }
    } 
/>
````
It will look something like this- 
![step1](https://user-images.githubusercontent.com/9073610/39581731-e0779e26-4f09-11e8-8ee6-b714369a6197.png)

2. Now when you will type google and select the first element appearing on the list. Even after selecting the element the list did not go away, the drop-down list remains if the word is matching. See-

![step2](https://user-images.githubusercontent.com/9073610/39581862-32ab08cc-4f0a-11e8-9039-afc6447c79b0.png)

  
**Expected behaviour:** [What you expect to happen]

What I want is that- If we have selected an item or element from the dropdown list, then the dropdown list should go away, which is not happening in the below example.

**Actual behaviour:** [What actually happens]

1. It will look something like this- 
![step1](https://user-images.githubusercontent.com/9073610/39581731-e0779e26-4f09-11e8-8ee6-b714369a6197.png)

2. Now when you will type google and select the first element appearing on the list. Even after selecting the element the list did not go away, the drop-down list remains if the word is matching. See-

![step2](https://user-images.githubusercontent.com/9073610/39581862-32ab08cc-4f0a-11e8-9039-afc6447c79b0.png)

  
### Versions
You can check directly on the website, here- [react-materialize]( https://react-materialize.github.io/#/forms)

The version I am using in my code is-
**react-materialize:** ^1.0.8
**materialize-css:** 0.100.1
**react:** 0.14.7

**Comment by alextrastero** [comment]
Thanks for spotting this @lovingyugs, we'll check it out ASAP

**Comment by lovingyugs** [comment]
@alextrastero Thanks for responding earlier. Just want to know that did you guys found out any solution for the issue? I am really looking forward to it.

**Comment by alextrastero** [comment]
Sorry no updates yet, do you want to give it a try??

**Comment by niross** [comment]
I'm happy to take this one if nobody else is.

**Comment by alextrastero** [comment]
@niross yes please :)

**Comment by alextrastero** [comment]
Fixed by @niross and published to 2.3.1
